AutoService.BLL/Infrastructure/InternalServiceException.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace AutoService.BLL.Infrastructure
+{
+    public class InternalServiceException : Exception
+    {
+        public string Property { get; protected set; }
```

Comment:
which purpose of this property?

it has no meaningfull name. Also empty string passed in all usages

Reply:
It can be used to pass the name of the property of model class, which is incorrect.

Reply:
Than it must have meaningful name and be used in code